Fix for Clarify Documentation for config_configuration_recorder #16575 (#16777)
hashicorp/terraform-provider-aws: website/docs/r/config_configuration_recorder.html.markdown

* `all_supported` - (Optional) Specifies whether AWS Config records configuration changes for every supported type of regional resource (which includes any new type that will become supported in the future). Conflicts with `resource_types`. Defaults to `true`.
* `include_global_resource_types` - (Optional) Specifies whether AWS Config includes all supported types of *global resources* with the resources that it records. Requires `all_supported = true`. Conflicts with `resource_types`.

> * `resource_types` - (Optional) A list that specifies the types of AWS resources for which AWS Config records configuration changes (for example, `AWS::EC2::Instance` or `AWS::CloudTrail::Trail`). See [relevant part of AWS Docs](http://docs.aws.amazon.com/config/latest/APIReference/API_ResourceIdentifier.html#config-Type-ResourceIdentifier-resourceType) for available types. Set `all_supported = false` is required to use this argument.

Reviewer: Hi @gunadhya! Thanks for the clarification, I'd like to suggest a small rewording, then we are good to go!

```suggestion
* `resource_types` - (Optional) A list that specifies the types of AWS resources for which AWS Config records configuration changes (for example, `AWS::EC2::Instance` or `AWS::CloudTrail::Trail`). See [relevant part of AWS Docs](http://docs.aws.amazon.com/config/latest/APIReference/API_ResourceIdentifier.html#config-Type-ResourceIdentifier-resourceType) for available types. In order to use this attribute, `all_supported` must be set to false.
```

Author: I've updated the statement. This is what is looks like in preview.

![image](https://user-images.githubusercontent.com/6939749/107742628-6d565200-6d35-11eb-9b46-af018d83da48.png)
